WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueFragment.java

```diff
@@ -336,7 +336,7 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
                 mImageManager.loadIntoCircle(mHeaderAvatar, ImageType.AVATAR_WITHOUT_BACKGROUND, mPhotoUrl);
             }
         } else {
-            mDialog = (FullScreenDialogFragment) getFragmentManager().findFragmentByTag(FullScreenDialogFragment.TAG);
+            mDialog = (FullScreenDialogFragment) getParentFragmentManager().findFragmentByTag(FullScreenDialogFragment.TAG);
```

Comment:
I've noticed a minor checkstyle issue on this line:
```
Line is longer than 120 characters (found 124). [LineLength]
```

Reply:
Oops looks like the checkstyle error might have went out of screen as well haha. Will fix it!

Reply:
Also i will be leaving test directions soon so I would request you to wait on this one until then.

Reply:
@antonis test instructions are up!

Reply:
Thank you for the heads up @07jasjeet 🙇 
My plan is to review the PR later today or tomorrow.